### PR TITLE
add the for example to 2nd line and remove the blank line and ;

### LINKED ISSUE #7139: [DOCS] Better formatting of text in the Events documentation.
### Description

Information in the Events documentation page is not formatted properly which may cause difficulty for the user to understand.
This small fix will bring more clarity in documentation.

### Screenshots

For Example can be written in the second line for better visibility 
Screenshot 2023-05-23 174216

Extra line spacing and a ";" can be seen
Screenshot 2023-05-23 174225

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jaskarandji**
## Fixes Issue

 Example: Closes #7139

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Set it

## Note to reviewers

If I did something wrong, please guide me through that. 
 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7234"></a>

### CODE REVIEW

File: pages/docs/how-to-guides/events.mdx [hunk 1]
```diff
@@ -73,15 +76,12 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
 | country           | false    | In which country, the event is hosting |
 
 4. Now you can commit your file and create a Pull Request. For more details please see <Link href="/docs/how-to-guides/editing">Editing Guide</Link>
-
-If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
-
-export default ({ children }) => (
-  <DocsLayout
+   If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
```
Comment (ChinmayMhatre) [inline pages/docs/how-to-guides/events.mdx:79]: Add a space here and fix the formatting with prettier

Comment (ChinmayMhatre) [reply pages/docs/how-to-guides/events.mdx:79]: I dont think you need to change the config just install prettier in vscode then right click, format with prettier

File: pages/docs/how-to-guides/events.mdx [hunk 2]
```diff
@@ -77,11 +77,6 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
...
-   export default ({ children }) => (
-   <DocsLayout
-    title="LinkFree Events Documentation"
-    section="Customising your Profile"
-    name="Events">
-    {children}
-    </DocsLayout>
-    );
+   
+   export default ({ children }) => (<DocsLayout  title="LinkFree Events Documentation" section="Customising your Profile" name="Events"> {children}</DocsLayout>);
```
Comment (jaskarandji) [inline pages/docs/how-to-guides/events.mdx:81]: Hi Chinmay,  I add the space to code but when I go to my repo it does not changed. It show  like this 👇🏿
code not change

File: pages/docs/how-to-guides/events.mdx [hunk 3]
```diff
@@ -14,13 +14,16 @@ All future events also appear on the Events page for the app.
 
 ### Add Event
 
-1. Create a folder that matches your GitHub username in the `data` directory and then in that folder create another folder called `events`. For example `data/eddiejaoude/events/`
+1. Create a folder that matches your GitHub username in the `data` directory and then in that folder create another folder called `events`.  
+   For example `data/eddiejaoude/events/`
```
Comment (SaraJaoude) [inline pages/docs/how-to-guides/events.mdx:18]: To match the style of other documentation:
```suggestion
   For example: `data/eddiejaoude/events/`
```

File: pages/docs/how-to-guides/events.mdx [hunk 4]
```diff
@@ -14,13 +14,16 @@ All future events also appear on the Events page for the app.
...
 
-2. In your folder add a `json` file with any name you like. We recommend putting the date first and then appending it with the event name. For example `2023-09-13-javascript-conference.json`.
+2. In your folder add a `json` file with any name you like. We recommend putting the date first and then appending it with the event name. 
+   For example `2023-09-13-javascript-conference.json`.
```
Comment (SaraJaoude) [inline pages/docs/how-to-guides/events.mdx:21]: To match the style of other documentation:
```suggestion
   For example: `2023-09-13-javascript-conference.json`.
```

File: pages/docs/how-to-guides/events.mdx [hunk 5]
```diff
@@ -73,15 +76,7 @@ _If you need help on how to edit this file, please see the <Link href="/docs/how
...
-export default ({ children }) => (
-  <DocsLayout
-    title="LinkFree Events Documentation"
-    section="Customising your Profile"
-    name="Events"
-  >
-    {children}
-  </DocsLayout>
-);
+   If you want to know how to properly add start, end, and CFP Close times, feel free to check [Unix Time stamp](https://www.unixtimestamp.com/).
```
Comment (SaraJaoude) [inline pages/docs/how-to-guides/events.mdx:79]:
```suggestion
```

File: pages/docs/how-to-guides/events.mdx (hunk 1 above)
Comment (jaskarandji) [reply pages/docs/how-to-guides/events.mdx]: Thanks Eddie and Sara for fixing the issue. It was my  first time for me  fixing the doc, Thanks to both of you and all the  folks who guided me.

**Pull request merged**